js/TxUtils.js

```diff
@@ -71,9 +71,8 @@ function checkSignature(signature) {
 
 function isValidOpenAliasAddress(address) {
     address = address.trim()
-    var dot = address.indexOf('.')
-    if (dot < 0)
-        return false
     // we can get an awful lot of valid domains, including non ASCII chars... accept anything
-    return true
+    // there should be something after the .
+    // make sure it is not some kind of floating number
+    return address.length > 2 && !parseFloat(address) && parseFloat(address) !== 0 && address.indexOf('.') >= 0
```

Comment:
Sorry, why parseFloat twice? I feel like I'm missing something obvious :P

Reply:
Hmm, iirc there was some interesting edge case here where normally the string would parse to 'NaN' which is falsy. Then something like `parseFloat("0...")` parses to 0 which should be falsy but falls through that first check? Idk let me double check I didn't really test this thoroughly.

Reply:
```suggestion
    return address.length > 2 && isNaN(parseFloat(address)) && address.indexOf('.') >= 0
```

Would this work too?

Reply:
Oh yeah, this looks much cleaner. Let me force push in a sec.